umh-core/pkg/fsm/dataflowcomponent/README.md

```diff
@@ -9,13 +9,31 @@ This package implements the Finite State Machine (FSM) for DataFlowComponent ser
 When multiple DataFlowComponent services are created, each with its own BenthosManager, they all independently allocate ports from the same default range (9000-9999). This can lead to port conflicts when different DataFlowComponent services try to use the same ports for their Benthos instances.
 
 ### Solution
-To avoid port conflicts, implement this strategy:
+To avoid port conflicts, implement one of these strategies:
 
-#### Create a single shared PortManager instance and pass it to all DataFlowComponent services:
+#### 1. Use Different Port Ranges (Simple but not recommended for production)
```

Comment:
this should not be introduced again, good example of why one should keep the commits "clean"